test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -170,8 +170,6 @@ public override void Setup()
                 workingDirectory = Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
             }
 
-            DynamoPathManager.PreloadAsmVersion("221", DynamoPathManager.Instance);
```

Comment:
@ikeough 
I think this is not required and have verified this with one test case. But just in case anything I have missed, please help double check this.